### ISSUE TITLE
Confusing info in "Open in Excel" pop-up

### ISSUE THREAD

**Issue by amnonig** [snuziale/vsts-open-in-excel#26]
The versions and even product (Team Foundation plug-in) are confusing in a number of ways:

- Seem outdated (specify 2017, when it's 2019)
- The link to "Visual Studio Office" plugin (the 2nd one) also brings me to the same page as the first one and I have to scroll it down a lot to find it (that's why I though initially that this was non-existent).

In short, there is less connection between the dialog and the page it links to that makes me feel as a user that something is wrong, compared to the 2 links getting me to the specific things to download or in another way have a tighter connection between the dialog and resulting page.

**Comment by NorthHighlandNicole** [comment]
Yes this documentation & link needs to be updated. Also why is there not a mac version of the Azure DevOps Office® Integration 2019 extension?
